scripts/ingests/sanghi23/Sanghi23-Ingest.py

```diff
@@ -0,0 +1,118 @@
+#evo_cols = ["name", "name_simbadable", "ra_j2000_formula", "dec_j2000_formula", "teff_evo", "teff_evo_err", "radius_evo", "radius_evo_err", "mass_evo", "mass_evo_err", "logg_evo", "logg_evo_err"]
+evo_table = pd.read_csv('FundamentalPropertiesbyEvoModel.csv')
+
+modeled_parameters_evo_ingest_dict = [
+    {'sourceName': row['name'],
+     'sourceAltName': row['name_simbadable'],
+     'ra': row['ra_j2000_formula'], 
+     'dec': row['dec_j2000_formula'],
+     'Radius': {'value': row['radius_evo'], 'value_error': row['radius_evo_err'], 'parameter': "radius", 'unit': 'R_jup', "model": "evolutionary"},
+     'log_g': {'value': row['logg_evo'], 'value_error': row['logg_evo_err'], 'parameter': "log g", 'unit': 'dex', "model": "evolutionary"},
+     'T_eff': {'value': row['teff_evo'], 'value_error': row['teff_evo_err'], 'parameter': "T eff", 'unit': 'K', "model": "evolutionary"},
+     'Mass': {'value': row['mass_evo'], 'value_error': row['mass_evo_err'], 'parameter': "mass", 'unit': 'M_jup', "model": "evolutionary"},
+     'reference': "Sang23"
+     } for _, row in evo_table.iterrows()
+]
+#atmo_cols = ["name", "name_simbadable", "ra_j2000_formula", "dec_j2000_formula", "teff_atmo", "radius_atmo", "logg_atmo", "log_lbol_lsun", "log_lbol_lsun_err"]
+atmo_table = pd.read_csv('FundamentalPropertiesbyAtmoModel.csv')
+modeled_parameters_atmo_ingest_dict = [
+    {'source': atmo["name"],
+     'ra': atmo["ra_j2000_formula"], 'dec': atmo["dec_j2000_formula"],
+     'Radius': {'value': atmo["radius_atmo"], 'parameter': "radius", 'unit': 'R_jup', "model": "atmospheric"},
+     'log_g': {'value': atmo["logg_atmo"], 'parameter': "log g", 'unit': 'dex', "model": "atmospheric"},
+     'T_eff': {'value': atmo["teff_atmo"], 'parameter': "T eff", 'unit': 'K', "model": "atmospheric"},
+     'log_L_bol': {'value': atmo["log_lbol_lsun"], 'value_error': atmo["log_lbol_lsun_err"], 'parameter': "L bol", 'unit': 'dex', "model": "atmospheric"},
+     'reference': "Sang23"  #confirm this
+     } for _, atmo in atmo_table.iterrows() 
+    ]
+
+def find_source_name(db, source_name, source_alt_name, ra, dec, reference):
+    found_source = find_source_in_db(db, source_name, ra=ra, dec=dec, ra_col_name = "ra", dec_col_name = "dec")
+    if len(found_source) > 0:
+        source_name = found_source[0]
+    
+    else:
+        print(f"Source {source_name} not found in database. Trying with Simbad name")
+        found_source = find_source_in_db(db, source_alt_name, ra=ra, dec=dec, ra_col_name = "ra", dec_col_name = "dec")
+        if len(found_source) > 0:
+            print(f"Found with simbad name {source_alt_name}. Ingesting alternative name.")
+            ingest_names(db, found_source[0], source_name)
+            source_name = found_source[0]
+        else:
+            print(f"Source {source_name} not found. Skipping for now.")
+            source_name = None
+
+
+    print(f'Found correct source name, saving as {source_name}')
+    return source_name
```

Comment:
I'm a bit confused about why you need to do this since you should have ingested all of the sources above.